README.md

```diff
@@ -117,12 +123,30 @@ Alternately, see [`ocamlformat.opam`](./ocamlformat.opam) for manual build instr
   (add-hook 'before-save-hook 'ocamlformat-before-save)))
 ```
 
+To change the options passed to OCamlFormat (to use the option `--disable-outside-detected-project` for example):
+- run `emacs`
+- run `M-x customize`
+- enter "ocamlformat" in the search field, and press the Search button
+- select the Ocamlformat Command item
+- enter your customized command in the Ocamlformat Command field, for example `ocamlformat --disable-outside-detected-project`
```

Comment:
Have you tested this, to make sure that this field is interpreted as a command rather than an executable name?

Reply:
You are right I added a choice option in the customization menu instead.